samcli/commands/check/command.py

```diff
@@ -0,0 +1,161 @@
+"""
+CLI command for "deploy" command
+"""
+import os
+
+import logging
+import functools
+
+import click
+
+from samcli.cli.main import pass_context, common_options as cli_framework_options, aws_creds_options, print_cmdline_args
+from samcli.cli.cli_config_file import TomlProvider, configuration_option
+from samcli.lib.telemetry.metric import track_command
+from samcli.lib.utils.version_checker import check_newer_version
+from samcli.commands._utils.options import template_option_without_build
+
+from samtranslator.translator.translator import Translator
+from samtranslator.public.exceptions import InvalidDocumentException
+
+import boto3
+from samtranslator.translator.managed_policy_translator import ManagedPolicyLoader
```

Comment:
Please move possible imports into cli implementation. Keeping imports here affects the latency of `--help`. Take this one for example; https://github.com/aws/aws-sam-cli/blob/develop/samcli/commands/deploy/command.py#L267